src/schema/comments.ts

```diff
@@ -211,6 +223,22 @@ export const typeDefs = /* GraphQL */ `
     OTHER
   }
 
+  type UserComment {
+    """
+    The user's vote for the comment
+    """
+    vote: Int!
+
+    """
+    Time when vote for the comment was last updated
+    """
+    votedAt: DateTime
```

Comment:
Does GQL actually need this?
We don't send/do anything with it on FE right?

Reply:
It was exposed for user post as well. But yeah, currently not used by FE. I'll check and remove it if not needed.